demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -546,6 +521,410 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
+                                     , const cv::GMatDesc &) {
+            return cv::empty_array_desc();
+        }
+    };
+
+    G_API_OP( AlignFacesForReidentification
+            , <cv::GMat( cv::GArray<cv::GMat>
+                       , cv::GArray<cv::Rect>
+                       , cv::GMat)>
+            , "custom.align_faces_for_reidentification") {
+        static cv::GMatDesc outMeta(const cv::GArrayDesc &,
+            const cv::GArrayDesc &,
+            const cv::GMatDesc & in) {
+            return in;
+        }
+    };
+
+    /** NOTE: Three crutch-Ops for empty {} GArray **/
+    G_API_OP( GetEmptyFaces
+            , <cv::GArray<detection::DetectedObject>(cv::GMat)>
+            , "custom.get_empty_faces_garray") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc &) {
+            return cv::empty_array_desc();
+        }
+    };
+
+    G_API_OP( GetEmptyActions
+            , <cv::GArray<DetectedAction>(cv::GMat)>
+            , "custom.get_empty_actions_garray") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc &) {
+            return cv::empty_array_desc();
+        }
+    };
+
+    G_API_OP( GetEmptyMatGArray
+            , <cv::GArray<cv::GMat>(cv::GMat)>
+            , "custom.get_empty_garray_of_mat") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc &) {
+            return cv::empty_array_desc();
+        }
+    };
+
+    /** OCV KERNELs */
+    GAPI_OCV_KERNEL(OCVFaceDetectorPostProc, FaceDetectorPostProc) {
+        static void run(const cv::Mat &in_ssd_result,
+                        const cv::Mat &in_frame,
+                        const detection::DetectorConfig & face_config,
+                        std::vector<detection::DetectedObject> &out_faces) {
+            std::unique_ptr<detection::FaceDetection> face_detector(new detection::FaceDetection(face_config));
+            out_faces = face_detector->fetchResults(in_ssd_result, in_frame);
+        }
+    };
+
+    GAPI_OCV_KERNEL(OCVGetRectFromImage, GetRectFromImage) {
+        static void run(const cv::Mat &in_image,
+                        std::vector<cv::Rect> &out_rects) {
+            out_rects.emplace_back(cv::Rect(0, 0, in_image.cols, in_image.rows));
+        }
+    };
+
+    GAPI_OCV_KERNEL(OCVGetRectsFromDetections, GetRectsFromDetections) {
+        static void run(const detection::DetectedObjects &detections,
+                        std::vector<cv::Rect> &out_rects) {
+            for (const auto& it : detections) {
+                out_rects.emplace_back(it.rect);
+            }
+        }
+    };
+
+    GAPI_OCV_KERNEL_ST(OCVGetRecognizeResult,  GetRecognizeResult, TrackersPack) {
+        static void setup(const cv::GArrayDesc &,
+                          const cv::GArrayDesc &,
+                          const cv::GArrayDesc &,
+                          const cv::GMatDesc &,
+                          const FaceRecognizerConfig &, std::shared_ptr<TrackersPack> &trackers,
```

Comment:
Please keep the alignment consistent. Here you have two arguments in the same line, while all other lines have a single argument